test/bdd/fixtures/agent-wasm/docker-compose.yml

```diff
@@ -69,12 +70,36 @@ services:
       - ARIESD_DEFAULT_LABEL=second-user-agent
       - BLOC_DOMAIN=testnet.trustbloc.local
       - WALLET_MEDIATOR_URL=${WALLET_ROUTER_URL}
+      - HTTP_SERVER_SDS_URL=https://localhost:8072/encrypted-data-vaults
     ports:
       - 8071:8071
     volumes:
       - ../keys/tls:/etc/tls
     command: start
 
+  couchdb.example.com:
+    container_name: couchdb.example.com
+    image: ${COUCHDB_IMAGE}:${COUCHDB_IMAGE_TAG}
+    ports:
+      - ${COUCHDB_PORT}:${COUCHDB_PORT}
+
+  sds.example.com:
```

Comment:
shouldn't this docker image depend of couch db docker image?

Ref: https://github.com/trustbloc/edge-sandbox/blob/master/test/bdd/fixtures/demo/docker-compose-demo-applications.yml#L43

Reply:
yes and no - when the EDV starts up, it doesn't communicate with the CouchDB server until a request is actually made to it... and there's a big `sleep` in the bdd-tests that makes everything wait for awhile to get started... I looked into the `depends_on` config option and it looks like it'll make the images start in an order based on that, but it won't actually wait until everything in that image is initialized... so it looks like we'll still need that wait anyway...

But it still seems like a good idea to add that `depends_on` config option... so I added it. I also made `user.agent.example.com` and `second.user.agent.example.com` depend on `sds.example.com`.